.github/workflows/integration-test.yaml

```diff
@@ -33,9 +33,11 @@ jobs:
       - name: Build packages
         run: pnpm run build:packages
 
-      - name: Run integration:xudt script
+      - name: Run integration:xudt/spore script
         working-directory: ./tests/rgbpp
-        run: pnpm run integration:xudt
+        run: |
+          pnpm run integration:xudt
+          pnpm run integration:spore
```

Comment:
If `integration:xudt` and `integration:spore` use different keys, they can run in prarallel, right?

**motivation**: To optimize the workflow and reduce the time consumed.

### Reference
- https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idstrategymatrix